protocol.md

```diff
@@ -1,58 +1,70 @@
 Magic Crystal
 =============
 
-HOWTO use magic-wormhole as a transport mechannism for Dark Crystal
-shards.
+_SB: Is this doc supposed to be a how-to guide for a (tech-savvy) user, or more a specification for a programmer to build an app?_
+
+This document provides a protocol and examples for using [Magic Wormhole](https://magic-wormhole.readthedocs.io/en/latest/welcome.html) as a transport mechanism for [secret sharing](https://darkcrystal.pw/what-is-secret-sharing/) with [Dark Crystal](https://darkcrystal.pw/) and the Public Key Infrastructure (PKI) of your choice.
```

Comment:
Add more here to make it more of an executive summary that is understandable to a non programmer.